ios/MarkdownLayoutManager.mm

```diff
@@ -7,9 +7,11 @@ - (void)drawBackgroundForGlyphRange:(NSRange)glyphsToShow atPoint:(CGPoint)origi
 
   [self enumerateLineFragmentsForGlyphRange:glyphsToShow usingBlock:^(CGRect rect, CGRect usedRect, NSTextContainer * _Nonnull textContainer, NSRange glyphRange, BOOL * _Nonnull stop) {
     __block BOOL isBlockquote = NO;
+    __block int currentDepth = 0;
```

Comment:
I believe we can move the declaration of `currentDepth` into `enumerateObjectsUsingBlock:`.
```suggestion
```